client/config_update_test.go

```diff
@@ -19,14 +19,14 @@ func TestConfigUpdateError(t *testing.T) {
 	)
 	assert.NilError(t, err)
 
-	err = client.ConfigUpdate(context.Background(), SwarmVersionedID{ID: "config_id", Version: swarm.Version{}}, swarm.ConfigSpec{})
+	_, err = client.ConfigUpdate(context.Background(), SwarmVersionedID{ID: "config_id", Version: swarm.Version{}}, ConfigUpdateOptions{Config: swarm.ConfigSpec{}})
 	assert.Check(t, is.ErrorType(err, cerrdefs.IsInternal))
 
-	err = client.ConfigUpdate(context.Background(), SwarmVersionedID{ID: "", Version: swarm.Version{}}, swarm.ConfigSpec{})
+	_, err = client.ConfigUpdate(context.Background(), SwarmVersionedID{ID: "", Version: swarm.Version{}}, ConfigUpdateOptions{Config: swarm.ConfigSpec{}})
```

Comment:
Same fir this one (and others below)

```suggestion
	_, err = client.ConfigUpdate(context.Background(), SwarmVersionedID{}, ConfigUpdateOptions{})
```